README.md

```diff
@@ -1,24 +1,57 @@
-# README
+# Budget App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> An application about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+---
 
-Things you may want to cover:
+## 🔗 [Live Demo](https://thawing-plains-76650.herokuapp.com/)
 
```

Comment:
No video recording of the project overview is found. Kindly endeavor to record a maximum of 5 minutes video presenting the features of the project. The recorded video should:
- [ ]  entail the **features** of the project.
- [ ] have a maximum length of **5 minutes**.
- [ ] be **clear and objective description** about the project in professional English. The topic and purpose of the project are made clear.
- [ ] contains a demo of the project, highlighting all the project features.
- [ ] contain a highlight about a piece of the code the student finds interesting or is proud of making.

Reply:
Thanks, I forgot about the video.